com.unity.ml-agents/Runtime/Academy.cs

```diff
@@ -306,13 +306,13 @@ void InitializeEnvironment()
 
         void ResetActions()
         {
-            DecideAction = () => {};
-            DestroyAction = () => {};
-            AgentSetStatus = i => {};
-            AgentSendState = () => {};
-            AgentAct = () => {};
-            AgentForceReset = () => {};
-            OnEnvironmentReset = () => {};
+            DecideAction = () => { };
```

Comment:
These were fixed in https://github.com/Unity-Technologies/ml-agents/pull/3305, please don't unfix them.

Reply:
Now un-un-fixing the fix